x/nft/blockchain/codec.proto

```diff
@@ -12,14 +13,26 @@ message BlockchainToken {
 }
 
 message TokenDetails {
-    repeated Network networks = 1 [(gogoproto.nullable) = false];
+    Config config = 1 [(gogoproto.nullable) = false];
 }
 
-message Network { // these are all placeholders
-    bytes tickers = 1;
-    bytes bootstrap = 2;
-    bytes codec = 3;
-    bytes codecSettings = 4;
+enum Codec {
+    bns = 0;
+    lisk = 1;
+}
+
+enum Env {
+    mainnet = 0;
+    testnet = 1;
+}
+
+message Config {
+    string chainID = 1;
+    Codec codec = 2;
+    string codecConfig = 3;
+    bool live = 4;
```

Comment:
q: what is live compared to Env? In the spec I found `liveness: <testnet, mainnet>`

Reply:
q: are you going to enforce mainnet => live (true), testnet => not live (false) ?

Reply:
@alpe I'm not sure how this is going to work yet, this will be defined in the spec from my understanding.

I guess the env is more of a tag than anything else, so it could indeed be converted to a string.
live: true/false is what would probably define "liveness".

Reply:
env is a tag for searching between production or non prod.
live indicates if the network is online and usable. Its possible a chain developer will want to issue their token for a mainnet before going live with it.